docs/modules/concepts/pages/storage/users.adoc

```diff
@@ -0,0 +1,59 @@
+= User Directory
+:navtitle: Users
+
+== Overview
+
+Whereas xref:storage/mailbox.adoc[mailboxes] are clearly a core component of
+any email system, user management is a lighter shade of gray.
+
+On the one hand, mailboxes ought to match up with actual users, so there is
+a clear relationship between mailbox management and user management.
+Because of this relationship, it is not at all strange to include a user directory
+directly within an email system.
```

Comment:
We need to mention that "user management" is abstracted with an API, and can be configured to follow an external user management tool.

IE there is an LDAP implementation for this.

Reply:
Yes, I agree. However, my intention for the "Concepts" section is to try to remain general, then have a "James specific" section later that is more specific to James.

Reply:
This information is indeed necessary. As a solution, I am going to leave the text as is here in the concepts, and instead include this information in the more technical section.